tests/tests_reboot.yml

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Ensure we can set crypto policy including reboot of the system
+  hosts: all
+  roles:
+    - role: linux-system-roles.crypto_policies
+      vars:
+        crypto_policies_policy: "FUTURE"
```

Comment:
You don't need double quotes here - in YAML everything is a string unless you use a non-numeric string which looks like a boolean value e.g. `foo: true` - YAML will interpret this as a bool, not a string

Reply:
Dropped these. Thanks.